angular/test/test-app/angular.json

```diff
@@ -127,12 +144,16 @@
                   "src": "src/environments/environment.ts",
                   "replaceWith": "src/environments/environment.prod.ts"
                 }
-              ]
+              ],
+              "optimization": true
             }
           }
         }
       }
     }
   },
-  "defaultProject": "test-app"
-}
+  "defaultProject": "test-app",
+  "cli": {
+    "analytics": "de9ff8d8-b1cf-40ad-9a28-80e7e8755d8e"
```

Comment:
Was this added intentionally?

Reply:
No, apologies. This is Angular CLI anonymous tracking (see https://angular.io/analytics) that is automatically offered when using CLI. Pusing a new commit now that disables it.